CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+In Development
```

Comment:
We'll take it for st2 `v3.2.0`

@m4dcoder I guess we'll release `v1.1.1` orquesta to include in st2 dependencies?

Reply:
Yes. I'll do a release of v1.1.1.